app/dashboard/notifications.py

```diff
@@ -223,3 +224,97 @@ def maybe_market_to_email(b, event_name, txid):
 
     return len(to_emails)
 
+def maybe_post_on_craigslist(bounty):
```

Comment:
`maybe_post_on_craigslist` looks good, but isn't called anywhere.  could you add a management command that posts any bounties with `web3_created` in the last hour to craigslist?

Reply:
Sure. I will also add that